src/transactions/transactionBuilder/transactionBuilder.ts

```diff
@@ -459,6 +461,29 @@ export function sign(args: { signer: Account; transaction: AnyRawTransaction }):
   }
 }
 
+/**
+ * Creates the and returns the Authenticator for passkey signed transactions.
+ *
+ * @param args.publicKey The public key of the passkey credential.
+ * @param args.signature The P256signature which is the signed challenge.
+ * @param args.authenticatorData The AuthenticatorData of the assertion.
+ * @param args.clientDataJSON The clientDataJSON of the assertion.
+ *
+ * @return The signer AccountAuthenticator
+ */
+export function getAuthenticatorForWebAuthn(args: {
+  publicKey: HexInput;
+  signature: HexInput;
+  authenticatorData: HexInput;
+  clientDataJSON: HexInput;
+}): AccountAuthenticator {
+  const { publicKey, signature, authenticatorData, clientDataJSON } = args;
+  const p256PublicKey = new P256PublicKey(publicKey);
+  const webAuthnSignature = new WebAuthnSignature(signature, authenticatorData, clientDataJSON);
+
+  return new AccountAuthenticatorSingleKey(new AnyPublicKey(p256PublicKey), new AnySignature(webAuthnSignature));
+}
```

Comment:
Perhaps make this a function on the authenticator instead as a constructor?  All I see here are things scoped to the authenticator by itself, vs needing to talk to anything else.